lib/Core/StatsTracker.cpp

```diff
@@ -259,6 +267,10 @@ StatsTracker::~StatsTracker() {
 }
 
 void StatsTracker::done() {
+  // Update statistics for reachability analysis
+  if (updateMinDistToUncovered && (statsFile || OutputIStats))
+      computeReachableUncovered();
+
   if (statsFile)
```

Comment:
Sometimes we ask about statsFile, other times about OutputStats, it would be nice to be consistent